tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -12,13 +14,13 @@
 
 
 @pytest.mark.xdist_group(name="sam_remote_invoke_single_lambda_resource")
-class TestSingleResourceInvoke(RemoteInvokeIntegBase):
+class TestSingleLambdaInvoke(RemoteInvokeIntegBase):
     template = Path("template-single-lambda.yaml")
 
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.stack_name = f"{TestSingleResourceInvoke.__name__}-{uuid.uuid4().hex}"
+        cls.stack_name = f"{TestSingleLambdaInvoke.__name__}-{uuid.uuid4().hex}"
```

Comment:
is it better to use `cls` instead of `TestSingleLambdaInvoke`? (considering the amount of code needed to change if you change the class name)
```suggestion
        cls.stack_name = f"{cls.__name__}-{uuid.uuid4().hex}"
```

Reply:
I can update the other test class names as well.